iree/turbine/kernel/wave/codegen/handlers.py

```diff
@@ -1062,7 +1161,7 @@ def handle_cast(emitter: WaveEmitter, node: fx.Node):
                 src_vector_type.element_type.width < dst_elem_type.width,
                 is_dst_float and is_src_float,
             )
-        ](dst_vector_type, vector_src)
+        ](dst_vector_type, vector_src, fastmath=arith_d.FastMathFlags.fast)
```

Comment:
ditto `fastmath=get_fast_math_flags(options)`

Reply:
@yichiche we're still missing the change for this to use the get_fast_math option btw